README.md

```diff
@@ -1,65 +0,0 @@
-# ARDemo project
```

Comment:
don't remove this file either, but simple put the steps to build and run using the CLI